packages/driver/cypress/integration/e2e/origin/origin.spec.ts

```diff
@@ -237,12 +247,13 @@ describe('cy.origin', () => {
           expect(err.message).not.to.include(`The following error originated from your application code, not from Cypress`)
           expect(err.codeFrame).to.exist
           expect(err.codeFrame!.frame).to.include('cy.origin')
+
           done()
         })
 
         const variable = () => {}
 
-        cy.origin('http://idp.com:3500', { args: variable }, (variable) => {
+        cy.origin('http://foobar.com:3500', { args: variable }, (variable) => {
```

Comment:
This spec would hang across all browsers when visiting idp.com; switching to foobar.com like all other specs in this file caused the test to pass (well, fail in the way we expect it too). This is odd, but not sure if it's indicative of a larger problem or the same problem recorded in https://github.com/cypress-io/cypress/issues/21300.